data_labeling_examples/bulk_labeling_python/config.py

```diff
@@ -0,0 +1,30 @@
+import multiprocessing as mp
+# for help, run:
+# python3 help.py
+
+# config file path
+CONFIG_FILE_PATH = "~/.oci/config"
+# config file profile
+CONFIG_PROFILE = "DEFAULT"
+# region identifier of DLS Dataset
+REGION_IDENTIFIER = "uk-london-1"
+# service_endpoint
+SERVICE_ENDPOINT_DP = f"https://dlstest-dp.{REGION_IDENTIFIER}.oci.oraclecloud.com"
+# compartment where DLS Dataset exists
+COMPARTMENT_ID = "ocid1.compartment.oc1..aaaaaaaamrnszmfhl7xzawtef3rl6gmsb62tcoag32yiv5ujorognkuzrqfq"
```

Comment:
Why do we need compartment Id as input from user ?

Reply:
in list records API we need compartment id and dataset id right?

Reply:
When we validate the datasetId provided by the user, we call get dataset, and in its response we get the compartmentId. Are we not validating user input ?